DOL.WHD.Section14c.Web/src/modules/pages/landingPageController.js

```diff
@@ -59,54 +42,66 @@ module.exports = function(ngModule) {
       }
     };
 
+    $scope.startNewApplication = function () {
+      if (stateService.ein) {
+        stateService.saveNewApplication().then(function(result) {
+            stateService.applicationId = result.data.ApplicationId;
+            $location.path('/section/assurances');
+            autoSaveService.start();
+          }).catch(function(error) {
+            $scope.handleApplicationLoadError(error.data);
+          });
+      }
+    };
+
     $scope.handleApplicationLoadError = function(message) {
       $scope.applicationLoadError.status = true;
       if(message) {
         $scope.applicationLoadError.message = message;
       }
     };
 
-    $scope.loadUserInfo = function(index) {
-      stateService.employerId = $scope.organizations[index].employer.id;
-      stateService.applicationId = $scope.organizations[index].applicationId;
-      stateService.ein = $scope.organizations[index].ein;
-      stateService.employerName = $scope.organizations[index].employer.legalName;
+    $scope.loadUserInfo = function(employerId, ein, employerName) {
+      stateService.employerId = employerId;
+      stateService.ein = ein;
+      stateService.employerName = employerName;
       return;
     }
 
     $scope.init = function () {
       apiService.userInfo(stateService.access_token).then(function(result) {
-        $scope.organizations = result.data.organizations;
-        $scope.applicationList = [];
         result.data.organizations.forEach(function(element) {
+          $scope.loadUserInfo(element.employer.id, element.employer.ein, element.employer.legalName);
```

Comment:
This sets the employer ID, etc., to match the last element in the organizations array, is that correct?  A few comments here to explain why would be helpful.

Reply:
I will include a comment, but my understanding that for milestone 1, we are not allowed the user to create multiple employers; the id, ein, and legalName will be the same for every organization in the array.